src/handlers/comment/handlers/comment-incentive.ts

```diff
@@ -0,0 +1,22 @@
+import { getBotContext, getLogger } from "../../../bindings";
+import { Payload } from "../../../types";
+
+export const commentIncentive = async (body: string) => {
+  const context = getBotContext();
+  const logger = getLogger();
+  const payload = context.payload as Payload;
+  const sender = payload.sender.login;
+
+  logger.debug(`Received '/comment-incentive' command from user: ${sender}`);
+
+  if (!payload.issue) {
+    logger.info(`Skipping '/comment-incentive' because of no issue instance`);
+    return `Skipping '/comment-incentive' because of no issue instance`;
+  }
+  const toggle = (body.includes("true") || body.includes("false")) && body.match(/@(\w+)/g);
```

Comment:
```suggestion
  const toggle = (body.includes("true") || body.includes("false")) && body.match(/@([\w-]+)/g);
```
usernames can have dashes `-`